packages/ag-charts-community/src/chart/interaction/tooltipManager.ts

```diff
@@ -106,4 +110,46 @@ export class TooltipManager {
 
         this.appliedState = { content: contentToApply, meta: metaToApply };
     }
+
+    public static makeTooltipMeta(
+        event: InteractionEvent<'hover'>,
+        canvas: HdpiCanvas,
+        datum: SeriesNodeDatum & Pick<ErrorBoundSeriesNodeDatum, 'yBar'>,
+        window: Window
+    ): TooltipMeta {
+        const { pageX, pageY, offsetX, offsetY } = event;
+        const position = {
+            xOffset: datum.series.tooltip.position.xOffset,
+            yOffset: datum.series.tooltip.position.yOffset,
+        };
+        const meta: TooltipMeta = {
+            pageX,
+            pageY,
+            offsetX,
+            offsetY,
+            event: event,
+            showArrow: datum.series.tooltip.showArrow,
+            position,
+        };
+
+        // On line and scatter series, the tooltip covers the top of errorbars when using
+        // datum.midPoint. Using datum.yBar.upperPoint renders the tooltip higher up.
+        const refPoint: Point | undefined = datum.yBar?.upperPoint ?? datum.midPoint;
```

Comment:
Note: see PR description for a preview of what the output looks like with `midPoint` and `upperPoint`